Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -580,13 +580,15 @@ class AtlasStepper {
   /// @param [in] state State that will be presented as @c BoundState
   /// @param [in] surface The surface to which we bind the state
   /// @param [in] transportCov Flag steering covariance transport
+  /// @param [in] doFreeToBoundCorrection Flag steering non-linearity correction during global to local transform
```

Comment:
as a simple naming, I think we usually do not call things `bool doFooCorrection` but rather `bool applyFooCorrection` or simply `bool correctFoo`.

Reply:
Changed to `freeToBoundCorrection` following the struct type `FreeToBoundCorrection`.